src/Attributes.php

```diff
@@ -2,13 +2,11 @@
 
 namespace FreezyBee\MojeId;
 
-use Nette\Object;
-
 /**
  * Class Attributes
  * @package FreezyBee\MojeId
  */
-class Attributes extends Object
```

Comment:
Je duvod proc se dalo pryc uplne extends Object a nepridalo use Nette\SmartObject; ?

Reply:
asi na to @pista5 zapomnel, tohle by byl BC break

Reply:
Jasne, na to jsem zapomnel